core/src/main/java/io/narayana/nta/logparsing/as8/filters/GetFilterKeywords.java

```diff
@@ -0,0 +1,39 @@
+package io.narayana.nta.logparsing.as8.filters;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class GetFilterKeywords {
+	
+	InputStream inputStream;
+	String filterKeywords;
+	 
+	public String getPropValues() {
+ 
+		try {
+			Properties prop = new Properties();
+
+			String propFileName = "filter.properties";			
+			inputStream = getClass().getClassLoader().getResourceAsStream(propFileName);
+
+			if (inputStream != null) {
+				prop.load(inputStream);
```

Comment:
Every time getPropValues() get's called you are re-reading the properties. How about just doing it once in a constructor.